src/modules/formBuilder/components/itemEditor/conditionals/EnableWhenCondition.tsx

```diff
@@ -27,11 +31,68 @@ interface EnableWhenConditionProps {
   enableWhenPath?: 'enableWhen' | `autofillValues.${number}.autofillWhen`; // path to enableWhen in form
 }
 
+const validOperatorsForType = (type: ItemType, repeats: boolean) => {
```

Comment:
should we try to capture this logic in the JSON Schema?

Reply:
Nice, I think this is a much better ux than before!

As for validating in the json schema or in the backend validator code, I think it is a good idea, but now that these restrictions are in place in the frontend interface, I also think it would be fine to leave as a todo/cleanup for later.

Reply:
Some validations and TODOs added to https://github.com/greenriver/hmis-warehouse/pull/4498